src/commands/schemaUpload.ts

```diff
@@ -297,20 +297,18 @@ export function extractDetail(message: string): string {
 /** Return the success message to show the user after having uploaded a new schema */
 export function schemaRegistrationMessage(
   subject: string,
-  existingVersion: number | undefined,
-  registeredVersion: number,
+  maxExistingVersion: number | undefined,
+  newlyRegisteredVersion: number,
 ): string {
-  // todo: make this return markdown and to not induce pain for the reader. Use a header
-  // that clearly delineates the different cases, then key / value rows for the details.
-
-  if (existingVersion === undefined) {
+  if (maxExistingVersion === undefined) {
     return `Schema registered to new subject "${subject}"`;
-  } else if (existingVersion === registeredVersion) {
+  } else if (maxExistingVersion >= newlyRegisteredVersion) {
```

Comment:
Better var names highlight the better comparator to use. The version assigned to the newly uploaded version can well be not the most recent version.